### ISSUE TITLE
Migration to organization

### ISSUE THREAD

**Issue by Neustradamus** [AdaLogics/go-fuzz-headers#96]
It is possible to migrate the personal account to organization?

Converting:
- https://help.github.com/articles/converting-a-user-into-an-organization/

It is possible to rename this account and create the organization and move/transfer the repository to the organization.

Rename account:
- https://help.github.com/en/enterprise/2.13/user/articles/changing-your-github-username

Create an organization:
- https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch

Transfer a repository:
- https://help.github.com/en/github/administering-a-repository/transferring-a-repository

Like:
- http://github.com/gnome
- http://github.com/kde
- ...

**Comment by AdamKorcz** [comment]
I am not sure what this is... I will need some elaboration if this is sincere.

**Comment by Neustradamus** [comment]
@AdamKorcz: @AdaLogics is a personal acount.

Please do not close this ticket before the solution.

**Comment by AdamKorcz** [comment]
I see. Why do we need to change the account type?

**Comment by Neustradamus** [comment]
@AdamKorcz: Personal account is for a real person, not a company.
And it is easy to see the difference.

**Comment by AdamKorcz** [comment]
> @AdamKorcz: Personal account is for a real person, not a company. And it is easy to see the difference.

Yes, that is correct, but that does not really answer my question. Why do we need to switch the account type?

**Comment by Neustradamus** [comment]
@AdamKorcz: It is easy: @AdaLogics is:
- A person?
- A company?

**Comment by AdamKorcz** [comment]
Yes, sure, but what is the issue using the library if it is in a personal Github account? I am not saying there isn't one btw. Just trying to understand the use case.

**Comment by Neustradamus** [comment]
@AdamKorcz: It will better to manage repositories if @AdaLogics becomes an organization.
It is free.
